src/main/java/game_world/objects/events/EncounterEvent.java

```diff
@@ -39,18 +41,33 @@ public EncounterEvent(String name, String encounterType, String npc) {
     public void preInput() {
         this.awaitInput = true;
         OutputHandler output = Output.getScreen();
-        // change text below
-        StringBuilder newMessage = new StringBuilder("[ENCOUNTER EVENT] What would you like to do?");
-        for (String input : inputs) {
-            newMessage.append("\n\t◈ ").append(input);
+        if (this.encounterType.equals("Encounter")) {
+            // change text below
+            StringBuilder newMessage = new StringBuilder("[ENCOUNTER EVENT] What would you like to do?");
+            for (String input : inputs) {
+                newMessage.append("\n\t◈ ").append(input);
+            }
+            output.generateText(String.valueOf(newMessage));
+        } else if (this.encounterType.equals("Quest")) {
```

Comment:
Single responsibility principle is violated here
Should be two different classes: EncounterEvent and QuestEvent